supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -10,8 +10,52 @@ Follow _The IBM Style Guide_  guidance of _day Month year_ for date formats, for
 When the format _day Month year_ causes a presentation or clarity issue, use instead: _Month day, year_ (for example, October 3, 2019).
 
 
-[[user-replaced-variables]]
-== User-replaced variables
+[[user-replaced-values]]
+== User-replaced values
+
+A _user-replaced value_, or _replaceable value_, is a value that the user must replace with a value that is relevant for their situation. User-replaced values are often found in places such as code blocks, file paths, and commands.
+
+Use descriptive names for user-replaced values and follow this general format: _<value_name>_.
+
+Ensure that user-replaced values have following characteristics:
```

Comment:
```suggestion
Ensure that user-replaced values have the following characteristics:
```

One more tiny suggestion to add 'the' back in.

Reply:
Whoops, good catch sorry!